text/0000-docs-team.md

```diff
@@ -0,0 +1,107 @@
+
+## Why documentation was left out
+
+Documentation was left out of the original list because it wasn't clear that there would be anyone but me on it. Furthermore,
+one of the original reasons for the subteams was to decide who gets counted amongst consensus for RFCs, but it was unclear
+how many documentation-related RFCs there would even be.
+
+## Chicken, meet egg
+
+However, RFCs are not only what subteams do. To quote the RFC:
+
+> * Shepherding RFCs for the subteam area. As always, that means (1) ensuring
+>   that stakeholders are aware of the RFC, (2) working to tease out various
+>   design tradeoffs and alternatives, and (3) helping build consensus.
+> * Accepting or rejecting RFCs in the subteam area.
+> * Setting policy on what changes in the subteam area require RFCs, and reviewing direct PRs for changes that do not require an RFC.
+> * Delegating reviewer rights for the subteam area. The ability to r+ is not limited to team members, and in fact earning r+ rights is a good stepping stone toward team membership. Each team should set reviewing policy, manage reviewing rights, and ensure that reviews take place in a timely manner. (Thanks to Nick Cameron for this suggestion.)
+
+The first two are about RFCs themselves, but the second two are more pertinent to documentation. In particular,
+deciding who gets `r+` rights is important. A lack of clarity in this area has been unfortuante, and has led to a
+chicken and egg situation: without a documentation team, it's unclear how to be more involved in working on Rust's
+documentation, but without people to be on the team, there's no reason to form a team. For this reason, I think
+a small initial team will break this logjam, and provide room for new contributors to grow.
+
+# Detailed design
+[design]: #detailed-design
+
+The Rust documentation team will be responsible for all of the things listed above. Specifically, they will pertain
+to these areas of the Rust project:
+
+* The standard library documentation
+* The book and other long-form docs
+* Cargo's documentation
+* The Error Index
+
+Furthermore, the documentation team will be available to help with ecosystem documentation, in a few ways. Firstly,
+in an advisory capacity: helping people who want better documentation for their crates to understand how to accomplish
+that goal. Furthermore, monitoring the overall ecosystem documentation, and identifying places where we could contribute
+and make a large impact for all Rustaceans. If the Rust project itself has wonderful docs, but the ecosystem has terrible
+docs, then people will still be frustrated with Rust's documentation situation, especially given our anti-batteries-included
+attitude. To be clear, this does not mean _owning_ the ecosystem docs, but rather working to contribute in more ways
+than just the Rust project itself.
+
+We will coordinate in the `#rust-docs` IRC room, and have regular meetings, as the team sees fit. Regular meetings will be
```

Comment:
Not sure how nitpicky you want to be here, but it might be good to specify the IRC server here